purs/src/App/Component/Image.purs

```diff
@@ -15,13 +14,15 @@ import Halogen.HTML.Properties as HP
   still loading, we indicate this. If the image failed to load, we increment a retry count
   and try again.
 -}
+data Query :: forall k. k -> Type
 data Query a
 
 data Action
   = LoadFailed
   | LoadSucceeded
   | RetryLoading
 
+type Slots :: forall k. Row k
```

Comment:
is this required?

Reply:
I'm not able to get rid of this constraint, without getting a warning.